mozci/task.py

```diff
@@ -451,6 +465,18 @@ def classifications(self):
             (t.classification, t.classification_note) for t in self.tasks if t.failed
         ]
 
+    @property
+    def durations(self):
+        return [t.duration for t in self.tasks]
```

Comment:
These should have corresponding abstract properties on the base `RunnableSummary` class.

Reply:
Added as part of latest push.